zerver/tests/test_subs.py

```diff
@@ -5187,9 +5187,9 @@ def test_users_getting_remove_peer_event(self) -> None:
         self.assertEqual(stream_delete_events[0]["users"], [user1.id])
         self.assertEqual(stream_delete_events[1]["users"], [user2.id])
         for event in stream_delete_events:
-            event_streams = event["event"]["streams"]
-            self.assert_length(event_streams, 1)
-            self.assertEqual(event_streams[0]["name"], "private_stream")
+            event_stream_ids = event["event"]["stream_ids"]
+            self.assert_length(event_stream_ids, 1)
+            self.assertEqual(event_stream_ids[0], private.id)
```

Comment:
Would be good to check `event["event"]["streams"]` field as well for now till we send it.

Reply:
Yes we send it but we would remove the `streams` field in future. So, we should make sure test checks whether the `streams` field is present and has the expected value.

Reply:
Aha, ok